src/common/components/nav/mobileNav.tpl.html

```diff
@@ -0,0 +1,158 @@
+<div class="mobile-nav-wrap" ng-class="{'on': $ctrl.mobileNavOpen}">
+  <div id="mobile-header">
+    <div id="mobile-logo"></div>
+
+    <div id="sign-in-section">
+      <sign-in-button id="sign-in"></sign-in-button>
+
+      <div id="mobile-nav-icon" ng-click="$ctrl.toggleMenu()" ng-class="{'open': $ctrl.mobileNavOpen}">
+        <span></span>
+        <span></span>
+        <span></span>
+        <span></span>
+      </div>
+    </div>
+  </div>
+
+  <div id="mobile-navigation" ng-class="{'open': $ctrl.mobileNavOpen, 'hidden': !$ctrl.mobileNavOpen}" ng-init="mobileTab = 'nav'">
+    <ul class="menu-icons nav nav-tabs" role="tablist">
+      <li role="presentation" ng-class="{'active': mobileTab === 'nav'}">
+        <a href="" ng-click="mobileTab = 'nav'; $ctrl.mobileMenuLevel[0] = ($ctrl.mobileMenuLevel[0] > 0 ? $ctrl.mobileMenuLevel[0] - 1 : 0);">
+          <img class="menu-icon-navigation"
+               ng-src="{{$ctrl.imgDomain + '/assets/img/' + ($ctrl.mobileMenuLevel[0] > 0 ? 'mobile-menu-back-icon.png' : 'mobile-menu-hamburger-icon.png')}}">
+        </a>
+      </li>
+      <li role="presentation" ng-class="{'active': mobileTab === 'search'}">
+        <a href="" ng-click="mobileTab = 'search'">
+          <img ng-src="{{$ctrl.imgDomain}}/assets/img/mobile-menu-search-icon.png">
+        </a>
+      </li>
+      <li role="presentation" ng-class="{'active': mobileTab === 'location'}">
+        <a href="" ng-click="mobileTab = 'location'">
+          <img ng-src="{{$ctrl.imgDomain}}/assets/img/mobile-menu-location-icon.png">
+        </a>
+      </li>
+      <li role="presentation" ng-class="{'active': mobileTab === 'global'}">
+        <a href="" ng-click="mobileTab = 'global'">
+          <img ng-src="{{$ctrl.imgDomain}}/assets/img/mobile-menu-globe-icon.png">
+        </a>
+      </li>
+      <li role="presentation" ng-class="{'active': mobileTab === 'cart'}">
+        <a href="" ng-click="mobileTab = 'cart'">
+          <img ng-src="{{$ctrl.imgDomain}}/assets/img/mobile-menu-cart-icon.png">
+        </a>
+      </li>
+      <li role="presentation" ng-class="{'active': mobileTab === 'more'}">
+        <a href="" ng-click="mobileTab = 'more'">
+          <img ng-src="{{$ctrl.imgDomain}}/assets/img/mobile-menu-ellipsis-icon.png">
+        </a>
+      </li>
+    </ul>
+
+
+    <div class="tab-content">
+      <div role="tabpanel" class="tab-pane active" ng-if="mobileTab === 'nav'">
+        <div class="main-menu-container">
+          <ul class="main-menu" id="main-menu" ng-style="{'left': (($ctrl.mobileMenuLevel[0] * -100) + '%')}">
+            <li ng-repeat="i in $ctrl.menuStructure" ng-class="{'has-children': i.children.length}" ng-if="!i.hideInNav">
```

Comment:
Could this be made into a component? It looks like it's hard coded to work only a specific number of levels, which might break if nav changes on AEM? You should be able to pass bindings of any information about current level. @OzzieOrca might have some ideas.